createAccountCore/src/main/java/com/hamdan/forzenbook/createaccountcore/di/CreateAccountModule.kt

```diff
@@ -55,4 +54,5 @@ object CreateAccountModule {
     }
 
     private const val MODULE_NAME = "create_account"
+    private const val LOGIN_BASE_URL = "https://forzen.dev/api/"
```

Comment:
Do you have duplicates of this?

Reply:
Yes I do, I was thinking for some reason this shouldn't go in the global constants, but now thinking about it again why shouldn't it.